build-packages/magento-scripts/lib/commands/start.js

```diff
@@ -5,6 +5,8 @@ const { Listr } = require('listr2');
 const { start } = require('../tasks/start');
 const pathExists = require('../util/path-exists');
 const { baseConfig } = require('../config');
+const googleAnalytics = require('@scandipwa/scandipwa-dev-utils/analytics');
```

Comment:
BTW, I still don't see analytics utility distributed in `dev-utilts` package, when do you think it will be released?